scripts/build

```diff
@@ -456,6 +461,31 @@ else
       echo "Executing (bootstrap): make $PKG_MAKE_OPTS_BOOTSTRAP" | tr -s " "
       make $PKG_MAKE_OPTS_BOOTSTRAP
       ;;
+
+    # python based build
+    "python2:host")
+      echo "Executing (host): python setup.py build" | tr -s " "
+      export LDSHARED="$CC -shared $LDSHARED"
+      python setup.py build --build-base .$HOST_NAME
```

Comment:
To leave room for python3

Reply:
Not really sure I follow... I'm referring to the fact you are using the unversioned `python` sym-link rather than the versioned `python2` sym-link - since this is a python2-specific change using the `python2` sym-link seems more logical/appropriate/correct. Also, you could/should use the full `$TOOLCHAIN/bin/python2` path and avoid using whatever version is found on `$PATH`.

Reply:
maybe a python as a synonym of python2?

Reply:
Thank you for reviewing @MilhouseVH 
I fixed to use the full `$TOOLCHAIN/bin/python2` path

Reply:
Sorry to be a pain but could you make the `$TOOLCHAIN/bin/python2` changes in the `echo "Executing (blah) python ..."` statements too?

Reply:
`echo` lines include python2 now

Reply:
I actually meant to include the `$TOOLCHAIN/bin/` part too, otherwise the command being echoed is not the same as the command being executed.

Reply:
I was not aware that these outputs were important :)

Reply:
Can be, if trying to reproduce or understand a particular failure - having the exact command that failed available in the log can help enormously. That's why these echo statements are there. :)

Reply:
I propose to leave this for another pull request: the other echo statements need to be updated and I do not know what level of detail you expect